update citations page (#456)
MDAnalysis/MDAnalysis.github.io: pages/citations.md

algorithms and libraries that were used in the program.

## MDAnalysis library ##

When using MDAnalysis in published work, please cite the following two
papers. 

(We are asking you to cite *both* papers if at all possible because
the 2016 paper describes many updates to the original 2011 paper and
neither paper on its own provides a comprehensive description of the
library. We will publish a complete self-contained paper with the

> upcoming 1.0 release of MDAnalysis, which will then supersede these

Reviewer: oops ... thanks for removing

Reviewer 2: Will 3.0 be the one? 😜

> To properly credit MDAnalysis, please cite both papers. The 2016 paper details significant updates to the 2011 paper, and together they provide a comprehensive description of the library.

Reviewer 2: ```suggestion
To properly credit MDAnalysis, please cite both the following papers. The 2016 paper details significant updates to the 2011 paper, and together they provide a comprehensive description of the library.
```
>

## MDAnalysis library ##

> To properly credit MDAnalysis, please cite both the following papers. The 2016 paper details significant updates to the 2011 paper, and together they provide a comprehensive description of the library.

Reviewer 3: either 'both of the following papers' or, as before, 'both papers'

## <a name="IncludedAlgorithms"></a>Included algorithms ##

The full list of citations for [included algorithms]({{site.pypi.docs}}/documentation_pages/references.html#citations-for-included-algorithms-and-modules) is maintained in our documentation and updated regularly.

## Acknowledgements

MDAnalysis was inspired by the Schulten Group’s [MDTools](http://www.ks.uiuc.edu/Development/MDTools/) for Python. The DCD reading code originates from VMD’s [catdcd](http://www.ks.uiuc.edu/Development/MDTools/catdcd/).

> MDAnalysis is released under the GPL license, with some third-party code under GPL-compatible licenses, such as the DCD reading code, which is licensed under the [UIUC Open Source

Reviewer 3: ```suggestion
MDAnalysis is released under the GPL license, with some third-party code under GPL-compatible licenses. For instance, the DCD reading code is licensed under the [UIUC Open Source
```

> MDAnalysis is released under the GPL license, with some third-party code under GPL-compatible licenses, such as the DCD reading code, which is licensed under the [UIUC Open Source

Reviewer: GPL -> LGPL

Perhaps link to a page where the license is described in more detail.

> Licence](http://www.ks.uiuc.edu/Development/MDTools/catdcd/license.html). See the files AUTHORS and LICENSE in the distribution for details.

Reviewer: Could you link to [AUTHORS](https://raw.githubusercontent.com/MDAnalysis/mdanalysis/refs/heads/develop/package/AUTHORS) and [LICENSE](https://raw.githubusercontent.com/MDAnalysis/mdanalysis/refs/heads/develop/package/LICENSE) ?

> The **MDAnalysis 'Atom' Logo** was designed by [**Christian

Reviewer: oops ... need to change this since NumFOCUS is now holding the rights and we are trademarking it and do not allow other uses (see https://github.com/MDAnalysis/branding ). Once we have the trademarks secured, we will state that here. Until then:

The **MDAnalysis 'Atom' Logo** was designed by [**Christian Beckstein**](https://www.linkedin.com/in/beckstein/). NumFOCUS as the fiscal sponsor of MDAnalysis holds all rights on the logo (see [branding/logos/LICENSE](https://raw.githubusercontent.com/MDAnalysis/branding/refs/heads/main/logos/LICENSE)).
